src/Microsoft.ML.Transforms/GcnTransform.cs

```diff
@@ -237,6 +246,32 @@ private static VersionInfo GetVersionInfo()
 
         private readonly ColInfoEx[] _exes;
 
+        /// <summary>
+        /// A helper method to create GlobalContrastNormalizer transform for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="name">Name of the output column.</param>
+        /// <param name="source">Name of the column to be transformed. If this is null '<paramref name="name"/>' will be used.</param>
+        /// <param name="subMean">Subtract mean from each value before normalizing.</param>
+        /// <param name="useStdDev">Normalize by standard deviation rather than L2 norm.</param>
+        /// <param name="scale">Scale features by this value.</param>
+        public static IDataTransform CreateGlobalContrastNormalizer(IHostEnvironment env, IDataView input, string name, string source = null, bool subMean = Defaults.GcnSubMean, bool useStdDev = Defaults.UseStdDev, Float scale = Defaults.Scale)
```

Comment:
>public static IDataTransform CreateGlobalContrastNormalizer(IHostEnvironment env, IDataView input, string name, string source = null, bool subMean = Defaults.GcnSubMean, bool useStdDev = Defaults.UseStdDev, Float scale = Defaults.Scale) [](start = 8, length = 236)

so so long

Reply:
Did some formatting.

---
In reply to: [198310144](https://github.com/dotnet/machinelearning/pull/405#discussion_r198310144) [](ancestors = 198310144)